package.json

```diff
@@ -5,21 +5,21 @@
   "main": "gulpfile.js",
   "dependencies": {
     "buble": "Noojuno/buble",
+    "clean-css-cli": "*",
```

Comment:
I'm not sure I understand why the CSS cleaner was moved along side JS tools.

If I could document this file I would, but the blank lines (seen below) were separating tools needed by different concepts: JS, CSS, SVG.

Reply:
Ah that change was unintentional I just ran `npm install stylelint --save`, and npm must of messed with the file.

Reply:
Okay I've restored the formatting